src/Components/Banner.js

```diff
@@ -1,33 +1,30 @@
 import React from 'react';
-import '../App.css';
+import { connect } from 'react-redux';
 
+import '../App.css';
 
+import Navigation from './Navigation';
+import AuthenticatedNavigation from './AuthenticatedNavigation';
 
-let Banner = () => {
-  return (
-    <div>
-      Banner has rendered
-    </div>
-  )
+let mapStateToProps = (state) => {
+  return {authenticated: state.isLoggedIn}
 }
 
-
-
-
-{/* class App extends Component {
-  render() {
+let Banner = ( {authenticated} ) => {
+  if (authenticated) {
```

Comment:
The start of authentication looks solid. Reminds me I need to add it in my code.